classes/helpers/user_helper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+// This file is part of the Accredible Certificate module for Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_accredible\helpers;
+
+use mod_accredible\apirest\apirest;
+use mod_accredible\local\credentials;
+
+/**
+ * Local functions related to credentials.
+ *
+ * @package    mod_accredible
+ * @subpackage accredible
+ * @copyright  Accredible <dev@accredible.com>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class user_helper {
```

Comment:
I think it would be better to not add a new subfolder like "helpers" as somebody is saying:

> If you are going to use a subfolder, then, to avoid clashes with future core Moodle class namespaces (e.g. the 'task' and 'event' subfolders), you should create your subfolders inside 'local', e.g. myplugin/classes/local/blah - after that, you're free to do what you want (and note that the use of 'local' is not enforced, just encouraged).

https://moodle.org/mod/forum/discuss.php?d=365379

As we're keeping our plugin local logic within `classes/local`, how about naming this class just `classes/local/users.php`?

Reply:
I have moved this class to `classes/local/users` as you recommend. Thanks for the explanation!